### ISSUE TITLE
Importer is unable to import pages with `u` tag in their html

### ISSUE THREAD

**Issue by sdmcraft** [adobe/helix-importer-ui#30]
### Expected Behaviour
Importer should be unable to import pages with `u` tag in their html

### Actual Behaviour
Importer is unable to import pages with `u` tag in their html

### Reproduce Scenario (including but not limited to)
1. Try to import this page https://research.alexanderforbes.com/chapters/objectives-governance-the-international-experience
2. The import fails with this error
```
import.ui.js:216 Error importing http://localhost:3001/chapters/objectives-governance-the-international-experience?host=https%3A%2F%2Fresearch.alexanderforbes.com: Cannot handle unknown node `element` Error: Cannot handle unknown node `element`
    at Object.ri (helix-importer.js:2:3931862)
    at Object.r [as handle] (helix-importer.js:2:3932476)
    at helix-importer.js:2:3932634
    at Compiler (helix-importer.js:2:3932794)
    at h.stringify (helix-importer.js:2:3846719)
    at helix-importer.js:2:4097403
    at Array.map (<anonymous>)
    at u (helix-importer.js:2:4097393)
    at Xt (helix-importer.js:2:3898757)
    at $t (helix-importer.js:2:3898355)
```

#### Platform and Version
MacOS Monetery v12.5.1
Helix importer Version v14.13.3

#### Workaround
Add this code while importing to replace `u` tags with `span`
```
document.querySelectorAll('u').forEach((u) => {
      const span = document.createElement('span');
      span.innerHTML = u.innerHTML;
      u.replaceWith(span);
    });
```

**Comment by kptdobe** [comment]
DOM like this one are the culprit:

```
<u><strong>Stage I: Governance structurePrinciple 1: Clear Roles and Responsibilities</strong></u>
```

**Comment by kptdobe** [comment]
Should be fixed with [v1.11](https://github.com/adobe/helix-importer-ui/commit/00b0b9d528c0c3a2f26127a38e4a545ed8118072)
